src/index.js

```diff
@@ -51,12 +53,13 @@ const ping = (options, shouldMute) => {
     // Fires all requests
     .then(response => {
       log(`Loading details of the latest ${options.limit} top stories`);
-
-      return fetchTopStoriesDetails(response, options.limit);
+      return fetchTopStoriesDetails(response);
     })
     // Returns a formatted array with the response request
     .then(response => {
-      return response.map(item => item.body);
+      const listItems = [];
```

Comment:
@erickbelfy Then I would add a new `then()` to the promise chain. I want to make them as atomic as possible to avoid confusion and to make tests easier. It should look like this:

```js
// (...)
// Returns a formatted array with the response request
.then(response => {
  return response.map(item => item.body);
})
// Sort the response if requested
.then(response => {
  return options.latest ? response.slice().sort(sortByTime) : response;
})
// (...)
```